app/grandchallenge/components/models.py

```diff
@@ -957,8 +974,7 @@ def _validate_value_only(self):
             )
 
     def _validate_value(self):
-        if self.interface.saved_in_object_store:
-            self._validate_file_only()
+        if self.interface.saved_in_object_store and self.file:
```

Comment:
Why has the `_validate_file_only` gone? Now you can get things that have a file and value.

Reply:
This is still unresolved.

Reply:
Sorry, I somehow thought I had changed that. Resolved now.